signale.d.ts

```diff
@@ -0,0 +1,148 @@
+// Type definitions for signale
+// Project: https://github.com/klauscfhq/signale
+// Definitions by: Resi Respati <https://github.com/resir014>, Kingdaro <https://github.com/kingdaro>, Joydip Roy <https://github.com/rjoydip>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.7
+
+/// <reference types="node" />
```

Comment:
This line could be removed, and we could add `@types/node` as a dependency/peerDependency.

Reply:
> including /// <reference types=" node" /> in a declaration file declares that this file uses names declared in @types/node/index.d.ts; and thus, this package needs to be included in the compilation along with the declaration file

Above statement has been taken from `typescript docs`. As per I understand if we keep this will haven't any problem in `compilation` time.

Reply:
I see, feel free to keep this line then.